src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -422,9 +422,28 @@
     },
     "http": {
       "displayName": "HTTP",
-      "description": "Register native handlers for http and setup Refit support",
+      "description": "Register native handlers for http.",
       "type": "parameter",
-      "datatype": "bool"
+      "datatype": "choice",
+      "defaultValue": "Default",
+      "choices": [
+        {
+          "choice": "default",
+          "description": "Native handlers",          
```

Comment:
```suggestion
          "description": "Native handlers",
```

Reply:
This should be a description similar to what we have here:
<img width="813" alt="image" src="https://github.com/user-attachments/assets/1486de28-8b98-4c7f-a572-1315bca85505" />

https://platform.uno/docs/articles/external/uno.extensions/doc/ExtensionsOverview.html

Something like "Enables configuration of native Http handlers and registration of basic endpoints"